.gitignore

```diff
@@ -0,0 +1 @@
+mypc/
```

Comment:
good. You may want to add a .vscode folder in here too and a few other tmp file types